protocol/tcp.go

```diff
@@ -77,3 +77,20 @@ func (t *TCP) UnmarshalBinary(data []byte) error {
 	return nil
 
 }
+
+// SplitData returns the TCP options and application layer message of the TCP packet.
```

Comment:
Do you think it is more convenient if you provide two functions like "func (t *TCP) OptionsInHeader() ([]byte, error)"  and "func (t *TCP) PayLoad() ([]byte, error)"